README.md

```diff
@@ -149,20 +148,24 @@ More about developing frontend [can be found here](frontend/README.md).
 
 ## Cypress tests
 
-Some [integration tests](https://github.com/Altinn/altinn-studio/tree/main/frontend/testing/cypress) for studio have
-been created with Cypress. More about these tests [can be found here](frontend/testing/cypress/README.md).
+Some [integration tests][17] for studio have been created with Cypress. More about these tests can be found [here](frontend/testing/cypress/README.md)
+
+## Playwright tests
+
+We're in the process of replaceing the Cypress framework with Playwright.
+More about these tests can be found [here](frontend/testing/playwright/README.md).
```

Comment:
Maybe add an extra sentence here what we use playwright tests for.

Reply:
I'll add another sentence or two